pages/coaching-dashboard.tsx

```diff
@@ -1,27 +1,36 @@
 import { useQuery } from "@apollo/client";
-import React from "react";
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
 import ProfileDetailCard from "../components/coding/studentPortal/ProfileDetailCard";
 import {
   FetchUserProfileCardResponse,
   FETCH_USER_PROFILE_CARD,
+  Users,
 } from "../graphql/fetchUserProfileCard";
 import { format } from "date-fns";
 import Link from "next/link";
+import {
+  skillifyUsersSelector,
+  setUserList,
+} from "../redux/skillifyUsersSlice";
 const coachingDashboard = () => {
+  const dispatch = useDispatch();
+  const userListState = useSelector(skillifyUsersSelector);
```

Comment:
follow the pattern in the line below to pull just userList from the useSelector hook... this is particularly helpful when you have a state object with multiple state variables